301-vm-32-data-disks-high-iops/StoragePool.ps1/StoragePool.ps1

```diff
@@ -0,0 +1,26 @@
+Configuration StoragePool
```

Comment:
@AshishSharma303 What does this script add to 301-vm-32-data-disks-high-iops template?